backend/src/v2/driver/driver.go

```diff
@@ -313,6 +313,12 @@ func makePodSpecPatch(
 		return "", fmt.Errorf("failed to make podSpecPatch: %w", err)
 	}
 
+	// Convert environment variables
+	userEnvVar := make([]k8score.EnvVar, 0)
+	for _, envVar := range container.Env {
```

Comment:
NIT: can we use `container.GetEnv()` provided by the proto compiler?

Reply:
This makes sense, since `container.GetEnv()` covers the case when container is nil. I will change the code. I don't think it's much a problem though, since the rest of the function also directly accessed the fields.